doc/userdoc/nest_server.rst

```diff
@@ -0,0 +1,649 @@
+that can be deployed either locally or on a remote machine. To give
+you a better idea of what NEST Server is good for, here are some of
+its main use cases.
+
+One scenario in which NEST Server comes in handy, is if you want to
+work on your laptop, but run your NEST simulations on a
+machine with higher performance or more memory, for instance, a big
+workstation or computer cluster at your lab. For this, you would
+deploy NEST Server on the remote machine, and use the provided
+:ref:`NEST Server Client <nest_server_client>` locally or write your
+own client using one of the recipes provided in the :ref:`section on
+advanced applications <nest_server_advanced>`.
+
+`NEST Desktop <https://nest-desktop.readthedocs.io/>`_, the web-based
+graphical user interface for NEST, uses NEST Server as its simulation
+backend. It supports server instances running either locally or
+remotely. More details about how to configure and run this setup can
+be found in the documentation of NEST Desktop.
+
+Last but not least, the latest version of the `HBP Neurorobotic
+Platform <https://neurorobotics.net/>`_ use the NEST Server to run the
+neuronal simulation as part of closed-loop robotic experiments. As it
+has rather specific requirements on the client side, it uses a custom
+NEST Client instead of the generic one shipped with NEST.
+
+If you yourself have an interesting situation in which you use NEST
+Server and would like to have it listed here, feel free to `drop us a
+line <https://github.com/nest/nest-simulator/issues>`_.
+
+Installation and execution
+--------------------------
+
+NEST Server is included in all source code distributions of NEST and
+consequently, also available in derived packages, our virtual
+machine, and Docker images.
+
+For native installations, the requirements can be simply installed via
+``pip``::
+
+  pip3 install RestrictedPython uwsgi flask flask-cors
+
+or by installing the full NEST development environment in case you
+prefer using ``conda``::
+
+  conda env create -f extras/conda-nest-simulator-dev.yml
+  conda acvitate nest-simulator
+
+As an alternative to a native installation, NEST Server is available
+from the NEST Docker image. Please check out the corresponding
+:ref:`installation instructions <docker_vm_install>` for more details.
+
+Run NEST Server
+~~~~~~~~~~~~~~~~~~~~~
+
+All NEST Server operations are managed using the ``nest-server``
+command that can either be run directly::
+
+  nest-server start
+
+or supplied to the execution command line for running the Docker
+container::
+
+  docker run -it --rm -e LOCAL_USER_ID=`id -u $USER` -p 5000:5000 nestsim/nest:latest nest-server start
+
+The generic invocation command line for the ``nest-server`` command
+looks as follows::
+
+  nest-server <command> [-d] [-o] [-h <host>] [-p <port>] [-u <user>]
+
+Possible commands are `start`, `stop`, `status`, or `log`. The meaning
+of the other arguments is as follows:
+
+-d
+    Daemonize the server process
```

Comment:
True. It may really be sounding weird to someone not deep into low-level Linux stuff. What about so?
```suggestion
    Run nest-server in the background (i.e. daemonize it)
```

Reply:
Looks good, just add a comma after i.e. (I see what you mean about the commas).